migration/migration-table.sql

```diff
@@ -5,7 +5,11 @@ $migration_table$
         _current_user_id oid = NULL;
         _ps_catalog_schema_owner_id oid = NULL;
         _migration_table_owner_id oid = NULL;
+        _old_search_path text;
     BEGIN
+        EXECUTE 'SHOW search_path' INTO STRICT _old_search_path;
+        SET search_path TO pg_catalog;
```

Comment:
why don't we do SET local here instead? That way we don't need to restore

Reply:
Same answer as for the transaction control. I don't think that `SET LOCAL` works that way.